src/widget/wtrackmenu.cpp

```diff
@@ -1059,29 +1059,31 @@ void WTrackMenu::slotClearAllMetadata() {
 }
 
 void WTrackMenu::slotShowTrackInfo() {
-    const auto trackPointers = getTrackPointers();
-    if (trackPointers.empty()) {
-        return;
-    }
```

Comment:
```
if (isEmpty()) {
    return;
}
```
would allow removing redundancy below

Reply:
Agreed. This additional checks upfront doesn't hurt and helps to keep the code readable.